projects/packages/forms/src/dashboard/class-dashboard.php

```diff
@@ -142,6 +142,7 @@ public function render_dashboard() {
 			'gdriveConnection'        => $jetpack_connected && Google_Drive::has_valid_connection( $user_id ),
 			'gdriveConnectURL'        => esc_url( Redirect::get_url( 'jetpack-forms-responses-connect' ) ),
 			'gdriveConnectSupportURL' => esc_url( Redirect::get_url( 'jetpack-support-contact-form-export' ) ),
+			'isWpcom'                 => function_exists( 'wpcom_site_has_feature' ),
```

Comment:
Since we use the `@automattic/jetpack-shared-extension-utils` package for the dashboard, maybe we could rely on `isSimpleSite` instead?
https://github.com/Automattic/jetpack/blob/3a3c4adc1c0913bac224ddb832713bd6101fe1ab/projects/js-packages/shared-extension-utils/src/site-type-utils.js#L14-L21

Reply:
The problem is we actually do want this on atomic too. Would `getSiteType()` work for that as well? I agree it'd be much easier on the eyes :)

Reply:
Yes, you can use `isAtomicSite` right below:

https://github.com/Automattic/jetpack/blob/3a3c4adc1c0913bac224ddb832713bd6101fe1ab/projects/js-packages/shared-extension-utils/src/site-type-utils.js#L23-L30

Reply:
Just a nit on this: I do love the simplicity of `function_exists( 'wpcom_site_has_feature' )` as a "is WPCOM" check, but this check is already on a couple (4 as far as I counted) places throughout the class. Do you think we should extract the check to some obvious `is_wpcom` method on the class and make all those checks rely on it? Then having some single point where the `function_exists( 'wpcom_site_has_feature' )` is seen and explained (albeit the obvious method name should have already spoiled what it does).
?

Reply:
Or, since you're already requiring the Status package for Forms, you could have a new is_wpcom method that would check for `( new Host() )->is_wpcom_simple()`?

Reply:
I updated the code to use `Host::is_wpcom_platform` on the backend and `isSimpleSite()` with `isAtomicSite()` on the frontend - removing the entry from our config object.